.github/workflows/sonar_analysis.yml

```diff
@@ -0,0 +1,28 @@
+name: Sonar Analysis
+
+# Controls when the workflow will run
+on:
+  push:
+    branches:
+      - main
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    name: Call Azure Build Pipeline
```

Comment:
call it `Call Azure Build Pipeline to SonarCloud check`

Reply:
-> Call Azure **Code Review** Pipeline to SonarCloud check